jumpscale/packages/marketplace/package.toml

```diff
@@ -1,5 +1,5 @@
 name = "marketplace"
-is_auth = true
+is_auth = false
```

Comment:
I think you can remove now the configuration of the package actors here
https://github.com/threefoldtech/js-sdk/blob/a03dcf35dfdf1b75611532056b47fb0658d8c450/jumpscale/packages/marketplace/package.py#L18